src/GoogleAdWordsEventForwarder.js

```diff
@@ -74,7 +74,7 @@
             if (window.google_trackConversion) {
                 window.google_trackConversion(adWordEvent);
             } else {
-                eventQueue.push(event);
+                eventQueue.push(adWordEvent);
```

Comment:
Was this intended to be part of a different PR?

Reply:
it was just something i caught when removing window.  i can make it a diff PR tho